contracts/registrar/ZNSSubRegistrar.sol

```diff
@@ -99,14 +105,30 @@ contract ZNSSubRegistrar is AAccessControlled, ARegistryWired, UUPSUpgradeable,
             !registry.exists(domainHash),
             "ZNSSubRegistrar: Subdomain already exists"
         );
+        require(
+            registry.exists(parentHash),
+            "ZNSSubRegistrar: Parent domain does not exist"
+        );
```

Comment:
is this really necessary?
In the below code, if the owner is 0x0, it will return false for everything and the address will also not match. this can possibly be avoided?

Reply:
re: above comment, removing